streams/src/main/java/org/apache/kafka/streams/processor/ExtractRecordMetadataTimestamp.java

```diff
@@ -50,15 +50,15 @@
      * Extracts the embedded metadata timestamp from the given {@link ConsumerRecord}.
      *
      * @param record a data record
-     * @param previousTimestamp the latest extracted valid timestamp of the current record's partition˙ (could be -1 if unknown)
+     * @param partitionTime the latest extracted valid timestamp of the current record's partition˙ (could be -1 if unknown)
```

Comment:
The explanation might need to be rephrased, too?

Reply:
Ack